config/generate/genesis/generate-kava-genesis.sh

```diff
@@ -264,6 +264,8 @@ add-eth-genesis-account-key bridge_relayer '.kava.users.bridge_relayer' 10000000
 add-genesis-account kava1cj7njkw2g9fqx4e768zc75dp9sks8u9znxrf0w 1000000000000ukava,1000000000000swp,1000000000000hard
 # swap module
 add-genesis-account kava1mfru9azs5nua2wxcd4sq64g5nt7nn4n8s2w8cu 5000000000ukava,200000000btcb,1000000000hard,5000000000swp,103000000000usdx
+# fractional balance test account
+add-genesis-account kava1w9vxuke5dz6hyza2j932qgmxltnfxwl78u920k "2ukava"
```

Comment:
Likely removing it soon after precisebank upgrade, but would be good to mention in the comment why it's 2ukava for clarity - must match total value of `evmutil.accounts[].balance`